web/src/hooks/useHover.ts

```diff
@@ -1,15 +1,21 @@
-import {useCallback, useState} from 'react';
+import {Dispatch, SetStateAction, useCallback, useState} from 'react';
 
-const useHover = () => {
+const useHover = (index: number, isAnyHovered: number[], setIsAnyHovered: Dispatch<SetStateAction<number[]>>) => {
   const [isHovering, setIsHovering] = useState(false);
 
   const onMouseEnter = useCallback(() => {
     setIsHovering(true);
-  }, []);
+    if (!isAnyHovered.includes(index)) {
```

Comment:
We need to find another way of forcing a hover state without having coupled functionality between this hook and the rest of the components.

Reply:
Agree with Oscar here. This logic feels coupled with the component.